src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -208,39 +207,36 @@ const JobHistory = _.flow(
               {
                 size: { basis: 150, grow: 0 },
                 headerRenderer: () => h(HeaderCell, ['Status']),
+                cellRenderer: ({ rowIndex }) => {
+                  const { workflowStatuses, status } = filteredSubmissions[rowIndex]
+                  return h(Fragment, [
+                    statusCell(workflowStatuses), _.keys(collapsedStatuses(workflowStatuses)).length === 1 && status
+                  ])
+                }
+              },
+              {
+                size: { basis: 220, grow: 0 },
+                headerRenderer: () => h(HeaderCell, ['Actions']),
                 cellRenderer: ({ rowIndex }) => {
                   const {
                     methodConfigurationNamespace, methodConfigurationName, submissionId, workflowStatuses,
                     status, submissionEntity
                   } = filteredSubmissions[rowIndex]
                   return h(Fragment, [
-                    statusCell(workflowStatuses), _.keys(collapsedStatuses(workflowStatuses)).length === 1 && status,
-                    (collapsedStatuses(workflowStatuses).running && status !== 'Aborting') && h(TooltipTrigger, {
-                      content: 'Abort all workflows'
-                    }, [
-                      h(Clickable, {
-                        onClick: () => this.setState({ aborting: submissionId })
-                      }, [
-                        icon('times-circle', { size: 20, style: { color: colors.green[0], marginLeft: '0.5rem' } })
-                      ])
-                    ]),
+                    (collapsedStatuses(workflowStatuses).running && status !== 'Aborting') && buttonPrimary({
+                      onClick: () => this.setState({ aborting: submissionId })
+                    }, ['Abort workflows']),
```

Comment:
This button overflows its cell on smaller screen widths. We should probably make this column a fixed width.